### ISSUE TITLE
Error: 400 Bad Request - follow up to issue #276

### ISSUE THREAD

**Issue by DMBlakeley** [hjdhjd/homebridge-myq#283]
**Describe The Problem:**
Continuously seeing error for last 12+ hours: `Error: 400 Bad Request`

iOS app working. myQ Twitter feed does not report any issues. Last problem was on 2/19/2021 and was resolved on the same day. Realize that you have put significant blood, sweat and tears into chasing changes on the myQ side but plugin is now not functioning.

**To Reproduce:**

Start homebridge with myQ plug-in. You will immediately see this error. Problem occurs on Homebridge v1.2.5 as well as v1.3.0.

**Logs:**

`[2/22/2021, 7:51:49 AM] [myQ] myQ API: https://partner-identity.myq-cloud.com/connect/authorize?client_id=IOS_CGI_MYQ&code_challenge=zRrYFUNnVS1XzsmFWcmJYF9R7lCP_oLMhubORqpZUgU&code_challenge_method=S256&redirect_uri=com.myqops%3A%2F%2Fios&response_type=code&scope=MyQ_Residential+offline_access Error: 400 Bad Request
[2/22/2021, 7:51:49 AM] [myQ] myQ API: Unable to access the OAuth authorization endpoint. Will retry later.`
```
Show the Homebridge logs here.
Remove any sensitive information.
```

**Homebridge Configuration:**

```json
            "email": "REDACTED",
            "password": "REDACTED",
            "debug": true,
            "platform": "myQ"

```

**Screenshots:**

**Environment:**

* **Homebridge Version:** v1.2.5 and v1.3.0
* **Node Version:** v14.15.5
* **Homebridge-myQ Plugin Version**: v2.6.2
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** iPhone 12 Pro / iOS 14.4
* **Operating System and OS Version:** macOS 11.2.1

**Comment by hjdhjd** [comment]
Duplicate of #276

**Comment by DMBlakeley** [comment]
Apologies for causing annoyance (again). You referenced Twitter feed on #276 and why I checked. Any guidance on how to get operational on my end?

**Comment by hjdhjd** [comment]
Time. :smile: And...not annoyed, but there's really not much to say that hasn't been said many times in the past.

I posted it because it's a rare acknowledgment of serious downtime. We've always known the infrastructure can be unreliable...this is the most unreliable we've seen it in years. I suspect because they are transitioning their infrastructure to a new provider, and trying to upgrade to a major new API revision. 

Yes - aware there are issues (and don't need more "me toos" :smile:). 
No - there's no quick / easy fix. 
No - unclear if there are net-new API revisions, or more infrastructure stability issues. 
No - there's no timeline.

TL;DR: We'll see what the next few days bring. For some of us, things seemed to improve in the past few hours. For others, it hasn't.

**Comment by DMBlakeley** [comment]
Ok, will be patient.
